functions/Write-DbaDataTable.ps1

```diff
@@ -300,6 +300,9 @@ function Write-DbaDataTable {
             #>
                 if ($PStoSQLTypes.Keys -contains $column.DataType) {
                     $sqlDataType = $PStoSQLTypes[$($column.DataType.toString())]
+                    if($column.MaxLength -gt 0 -and ($column.DataType -in ("String", "System.String"))) {
```

Comment:
Please wrap this up to only be run if switch parameter is passed in.

Parameter name like `-UseDynamicStringLength`

We generally use an internal function to handle testing if a parameter was passed in...

```powershell
if (Test-Bound 'UseDynamicStringLength') {....
```